mentat/commands.py

```diff
@@ -241,6 +242,42 @@ def help_message(cls) -> str:
         return "Clear the current conversation's message history"
 
 
+SEARCH_RESULT_BATCH_SIZE = 10
+
+
+class SearchCommand(Command, command_name="search"):
+    async def apply(self, *args: str) -> None:
+        stream = SESSION_STREAM.get()
+        code_context = CODE_CONTEXT.get()
+
+        if len(args) == 0:
+            await stream.send("No search query specified\n", color="yellow")
+            return
+        try:
+            query = " ".join(args)
+            results = await code_context.search(query=query)
+        except UserError as e:
+            await stream.send(str(e), color="red")
+            return
+
+        for i, (feature, score) in enumerate(results):
+            _i = f"{i}: " if i < 10 else f"{i}:"
+            await stream.send(f"{_i} {score:.3f} | {feature.path}")
+            if i > 0 and i % SEARCH_RESULT_BATCH_SIZE == 0:
+                await stream.send("\nShow More results? ")
+                if not await ask_yes_no(default_yes=True):
+                    break
+        await stream.send("Search complete", color="green")
```

Comment:
I think from a UX perspective it's better not to send this message. The user will know the search is over. We don't send a similar message when they ask a general question and the model responds.